### ISSUE TITLE
Logging:  [Server thread/INFO]: [STDOUT]: []

### ISSUE THREAD

**Issue by TheWyzer** [Chuunkyz/Benchmark-1.17.1#4]
I did try to Goo this, all I could find is that it did occur in MC back in 2017, but it had coordinates, and was resolved.  Other sites for other "programs", not MC indicate it is a debug log "left on".

This fills the log up within seconds, as there are about 60 entries per second.

`[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []
[10:28:39] [Server thread/INFO]: [STDOUT]: []`

**Comment by TauzentBlitz** [comment]
FIX: Update Spatial harvesters from 0.11.12 to 0.11.12a.

I get the same thing.
I downloaded [badstdout](https://modrinth.com/mod/badstdout) and put it in the modpack. I'm gonna try running it again and see if that makes things a bit more clear.

Looks like it's spatial harvesters causing the logs.

Here's the log:
```
[18:36:06] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:06] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:06] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:06] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:06] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:06] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:06] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:06] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:06] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:06] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:06] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:06] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:06] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:06] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:06] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:07] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:07] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:07] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:07] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:07] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:07] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:07] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:07] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:07] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:07] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:07] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:07] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:07] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:07] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:07] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:07] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:07] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:07] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:07] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
[18:36:07] [Server thread/INFO]: [dunkmania101.spatialharvesters.data.ChunkLoaderData] []
```

**Comment by Chuunkyz** [comment]
fixed ! (spatial harvesters released a new version)
